sci-libs/pdal/pdal-2.9.0-r1.ebuild

```diff
@@ -18,6 +18,7 @@ IUSE="debug postgres test"
 BDEPEND="
 	sys-devel/gettext
 	virtual/pkgconfig
+	test? ( dev-cpp/gtest )
```

Comment:
https://github.com/PDAL/PDAL/blob/master/cmake/gtest.cmake has:

```cmake
set(MIN_GTest_VERSION "1.15.0")
```

Installing gtest was not sufficient for me -- I had the stable version of gtest, so pdal still errored out from trying to fetch a newer version. Please add a minimum bound.

Reply:
And filed https://bugs.gentoo.org/961469